src/main/java/org/snomed/snowstorm/core/data/services/AxiomConversionService.java

```diff
@@ -115,11 +115,7 @@ private AxiomRelationshipConversionService setupConversionService(String branchP
 					internalRelationship.setSourceId(sourceId != null ? sourceId.toString() : null);
 					internalRelationship.setGroupId(externalRelationship.getGroup());
 					internalRelationship.setInferred(false);
-
-					final org.snomed.otf.owltoolkit.domain.Relationship.ConcreteValue externalConcreteValue = externalRelationship.getValue();
-					if (externalConcreteValue != null) {
-						internalRelationship.setConcreteValue(externalConcreteValue.getRF2Value());
-					}
+					internalRelationship.setConcreteValueFromExternal(externalRelationship.getValue());
```

Comment:
To avoid having two ConcreteValues in the Relationship class you could have a method like this setConcreteValue(String type, String value).

Reply:
I have made those changes.